version.bzl

```diff
@@ -1,3 +1,10 @@
+# Minimum supported Bazel version.  Should match .bazelminversion file and
+# https://buildkite.com/kythe-project/bazel-minimum/settings.
+MIN_VERSION = "0.25.1"
+
+# Maximum supported Bazel version.  Should match .bazelversion file.
+MAX_VERSION = "0.26.0"
```

Comment:
As an aside, I see that 0.26.1 got released, but I haven't tried it yet.

Reply:
Updated separately in https://github.com/kythe/kythe/pull/3797